src/utils/wallet/providers/okx_wallet.ts

```diff
@@ -176,23 +175,46 @@ export class OKXWallet extends WalletProvider {
   };
 
   // Inscriptions are only available on OKX Wallet BTC mainnet (i.e okxWallet.bitcoin)
```

Comment:
Should we add a check on whether we are on `mainnet` or not? If not, we should throw an error or something.